README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
> > Am I right in thinking that there could be a class AmateurFunctor with the opposite problem?

> In something like Haskell, a type that implements AmateurFunctor could only be declared as Contravariant.

Yup, that's the sort of thing I was thinking.

I think I get it now. It's for the same reason that the functor for a tuple is on the second element and for a function is on the codomain?

One thing I'm slightly confused about is the variance of type arguments. I think that in a value level function `f :: a -> b -> c`, both of the first two params are in negative position. Is that different for a kind `T :: * -> * -> *`? Or am I misapplying reasoning in an inappropriate domain?